backend/src/packages/chats/chat.controller.ts

```diff
@@ -302,6 +331,55 @@ class ChatController extends BaseController {
       ),
     };
   }
+
+  /**
+   * @swagger
+   * /chats/{id}:
+   *   delete:
+   *     description: Delete chat by id
+   *     security:
+   *      - bearerAuth: []
```

Comment:
Let's also add path parameter
```suggestion
   * /chats/{id}:
   *   delete:
   *     description: Delete chat by id
   *     security:
   *      - bearerAuth: []
   *     parameters:
   *       -  in: path
   *          description: Chat id
   *          name: id
   *          required: true
   *          type: number
   *          minimum: 1
```